resin/models/config.py

```diff
@@ -41,6 +50,7 @@ def get_all(self):
         if self._config is None:
             self._config = self.base_request.request(
                 'config', 'GET', endpoint=self.settings.get('api_endpoint'))
+            self._config['deviceTypes'] = map(self._patch_device_type, self._config['deviceTypes'])
```

Comment:
This is a bit fragile -- I think it'll break in Python 3. Better move ``_patch_device_type`` to a module-level function instead of method, and reference it here instead.